flowmachine/tests/test_raster_statistics.py

```diff
@@ -5,106 +5,92 @@
 """
 Unit tests for the RasterStatistics utilities.
 """
-import pandas as pd
 
-from unittest import TestCase
 import pytest
 
 from flowmachine.core import Table
 from flowmachine.features.raster import RasterStatistics
 
 
-class RasterStatisticsTestCase(TestCase):
+def test_computes_expected_clipping_values(get_dataframe):
     """
-    Series of tests for the RasterStatistics() utility
-    methods.
+    RasterStatistics() returns correct values when clipping vector and raster layers.
     """
+    G = "admin2pcod"
+    vector = Table(schema="geography", name="admin2")
+    r = RasterStatistics(
+        raster="population.worldpop_2015_un", vector=vector, grouping_element=G
```

Comment:
I'd very much prefer to not call this table `worldpop_2015_un` (since it's now just a test raster). Also, maybe we should move this raster into a specific `testing` schema as opposed to keeping it in the `population` schema that's meant for real data? In any case, I'm happy to defer these changes (if they are desired) to a separate PR.